js_modules/dagster-ui/packages/ui-core/src/overview/OverviewResourcesRoot.tsx

```diff
@@ -44,17 +50,28 @@ export const OverviewResourcesRoot = () => {
       notifyOnNetworkStatusChange: true,
     },
   );
-  const {data, loading} = queryResultOverview;
-  useBlockTraceOnQueryResult(queryResultOverview, 'OverviewResourcesQuery');
+  const {data, loading: queryLoading} = queryResultOverview;
+  useBlockTraceUntilTrue('OverviewResources', !!data || !workspaceLoading);
   const refreshState = useQueryRefreshAtInterval(queryResultOverview, FIFTEEN_SECONDS);
 
   // Batch up the data and bucket by repo.
   const repoBuckets = useMemo(() => {
     const visibleKeys = visibleRepoKeys(visibleRepos);
-    return buildBuckets(data).filter(({repoAddress}) =>
+    const cachedEntries = Object.values(cachedData).filter(
+      (location): location is Extract<typeof location, {__typename: 'WorkspaceLocationEntry'}> =>
+        location.__typename === 'WorkspaceLocationEntry',
```

Comment:
Does this cached data really contain everything needed for this query?  `ResourceEntryFragment` is the meat of what we're after here, and isn't referenced from the WorkspaceContext's `WorkspaceRepository` query 

(at least, not on master... maybe it is now? :-o )